webpack/components/OpenscapRemediationWizard/steps/ReviewHosts.js

```diff
@@ -0,0 +1,199 @@
+      setParamsAndAPI({ ...params, search: newSearch, page: 1 });
+    }
+  };
+
+  const { pageRowCount } = getPageStats({
+    total: subtotalCount,
+    page,
+    perPage,
+  });
+  const { fetchBulkParams, ...selectAllOptions } = useBulkSelect({
+    results,
+    metadata: { total: subtotalCount, page },
+    initialSearchQuery: apiSearchQuery || defaultSearch,
+    isSelectable: () => true,
+    initialArry: [hostId],
+  });
+  const {
+    selectPage,
+    selectedCount,
+    selectOne,
+    areAllRowsOnPageSelected,
+    areAllRowsSelected,
+    isSelected,
+  } = selectAllOptions;
+
+  useEffect(() => {
+    if (selectedCount) setHostIdsParam(fetchBulkParams());
+  }, [selectedCount, fetchBulkParams, setHostIdsParam]);
+
+  const selectionToolbar = (
+    <ToolbarItem key="selectAll">
+      <SelectAllCheckbox
+        {...{
+          selectAll: noop, // I don't think it really can select all since ids from other pages are still need to be loaded/fetched
+          selectPage,
+          selectNone: noop, // we shouldn't allow to select none since it migth lead to selecting a wrong host
+          selectedCount,
+          pageRowCount,
+        }}
+        totalCount={subtotalCount}
+        areAllRowsOnPageSelected={areAllRowsOnPageSelected()}
+        areAllRowsSelected={areAllRowsSelected()}
+      />
+    </ToolbarItem>
+  );
+
+  const RowSelectTd = ({ rowData }) => (
+    <Td
+      select={{
+        rowIndex: rowData.id,
+        onSelect: (_event, isSelecting) => {
+          selectOne(isSelecting, rowData.id);
+        },
+        isSelected: rowData.id === hostId || isSelected(rowData.id),
+        disable: rowData.id === hostId || false,
+      }}
+    />
+  );
+  RowSelectTd.propTypes = {
+    rowData: PropTypes.object.isRequired,
+  };
+
+  const columns = {
+    name: {
+      title: __('Name'),
+      wrapper: ({ id, name }) => <a href={foremanUrl(`hosts/${id}`)}>{name}</a>,
+      isSorted: true,
+    },
+    operatingsystem_name: {
+      title: __('OS'),
+    },
+  };
+
+  return (
+    <>
+      <WizardHeader
+        title={__('Review hosts')}
+        description={__(
+          'The remediation will be applied to the current host by default. Here you can select additional hosts.'
+        )}
+      />
+      <Toolbar ouiaId="table-toolbar" className="table-toolbar">
+        <ToolbarContent>
+          <ToolbarGroup>
+            {selectionToolbar}
+            <ToolbarItem className="toolbar-search">
+              <SearchBar
+                data={searchProps}
+                initialQuery={apiSearchQuery}
+                onSearch={onSearch}
+              />
+            </ToolbarItem>
+            {status === STATUS.PENDING && (
+              <ToolbarItem>
+                <Spinner size="sm" />
+              </ToolbarItem>
+            )}
+          </ToolbarGroup>
+        </ToolbarContent>
+      </Toolbar>
+      <Table
+        isEmbedded
+        isActionable={false}
```

Comment:
isActionable is not a prop, fixed the empty actions here: https://github.com/theforeman/foreman/pull/10082